server/app/fixtures.py

```diff
@@ -1,37 +1,25 @@
-from .models import Users, Profiles, Tasks, Bids, Payments
-from django.contrib.auth.hashers import make_password
+from .models import Users, Tasks, Bids, Payments
 from django.utils.timezone import now, timedelta
 
 
-# Empty database before populating mock data
-def delete_mock_data():
-    if Users.objects.exists():
-        Users.objects.all().delete()
-        Profiles.objects.all().delete()
-        Tasks.objects.all().delete()
-        Bids.objects.all().delete()
-        Payments.objects.all().delete()
-        print("Mock data deleted.")
-    else:
-        print("No mock data to delete.")
-
-
 # Create user instances
 def create_users():
     if not Users.objects.exists():
         user1 = Users.objects.create(
-            email='user1@example.com',
-            mobile='1234567890',
-            password_hash=make_password('password1'),
-            status='active',
-            user_role='poster'
+            email="bidder@example.com",
+            username="bidder_test",
+            mobile="1234567890",
+            password="password1",
+            is_bidder=True,
+            is_poster=False,
         )
```

Comment:
Not specific to here I guess - is there ever a page that asks for both `email` and `mobile`? Email and phone is only used as `username` so I don't think there are cases where both `email` and `mobile` are filled out, only one of them

Reply:
just mock data so, nvm this (username and mobile both have default value so it's easy to scale up when is necessary)